src/content/quickstart/openshift/_index.md

```diff
@@ -137,10 +141,44 @@ subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --cluster-id
 subctl join --kubeconfig cluster-b/auth/kubeconfig broker-info.subm --cluster-id cluster-b
 ```
 
-### Verify connectivity
+#### Verify connectivity
 
 This will run a series of E2E tests to verify proper connectivity between the cluster pods and services
 
 ```bash
 subctl verify-connectivity cluster-a/auth/kubeconfig cluster-b/auth/kubeconfig --verbose
 ```
+
+### Install Submariner with Service Discovery
+
+{{% notice info %}}
+
+The Lighthouse project is meant only to be used as a development preview. Installing the operator on an Openshift cluster may disable some of the operator features.
```

Comment:
I'm not clear on the second sentence - what features may be disabled and why and why only on Openshift?

Reply:
We disable Openshift CVO , it could have multiple side effects, do you think mentioning that is worth. This warning is just to use lighthouse in dev environment only.

Reply:
I think if we're going to mention that it may have side effects with Openshift we should specifically outline what may/would be affected and what that might mean to the user. Not being familiar with Openshift and the possible side effects myself, I did find the second statement a bit vague. As such, it's hard for me to say if we should omit it.

Reply:
Done.